### PR TITLE
fix(App): add ui system library version in env file

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add new variable in `env` file to set doppler ui library version

### CODE REVIEW

**Comment by andresmoschini** [comment]
Is it possible apply the same environment variable into index.html?

We have inconsistencies right now:

https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L17

https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L80

**Comment by fcoronelmakingsense** [comment]
> Is it possible apply the same environment variable into index.html?
> 
> We have inconsistencies right now:
> 
> https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L17
> 
> https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L80

How? because html it's outside react

**Comment by andresmoschini** [comment]
> How? because html it's outside react

I do not know. Another option is adding a  kind of automated test to ensure consistency.

**Comment by cbernat** [comment]
@fcoronelmakingsense you could use this env variable in the html too, like this https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L19

**Comment by andresmoschini** [comment]
> @fcoronelmakingsense you could use this env variable in the html too, like this https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L19

👏 👏 👏

**Comment by fcoronelmakingsense** [comment]
> @fcoronelmakingsense you could use this env variable in the html too, like this https://github.com/FromDoppler/doppler-webapp/blob/master/public/index.html#L19

Thanks y already apply!

File: .env [hunk 1]
```diff
@@ -13,3 +13,4 @@ REACT_APP_USE_DOPPLER_LEGACY_SIGNUP=false
 REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 NODE_ICU_DATA=node_modules/full-icu
 REACT_APP_ROUTER=browser
+REACT_APP_DOPPLER_UI_LIBRARY_VERSION = 3.51.5
```
Comment (andresmoschini) [inline .env:16]: What do you think about including the `v` in the version value?

```suggestion
REACT_APP_DOPPLER_UI_LIBRARY_VERSION = v3.51.5
```

Comment (andresmoschini) [reply .env:16]: Another option could be this:

```suggestion
REACT_APP_DOPPLER_UI_BASE_URL = https://cdn.fromdoppler.com/doppler-ui-library/v3.51.5
```

**Comment by andresmoschini** [comment]
🚀

**Pull request merged**